src/Aspire.ProjectTemplates/Aspire.ProjectTemplates.csproj

```diff
@@ -57,8 +57,8 @@
                       Lines="$([System.IO.File]::ReadAllText('%(TemplateProjectFiles.FullPath)')
                                                  .Replace('!!REPLACE_WITH_LATEST_VERSION!!', '$(PackageVersion)')
                                                  .Replace('!!REPLACE_WITH_ASPNETCORE_9_VERSION!!', '$(MicrosoftAspNetCorePackageVersionForNet9)')
-                                                 .Replace('!!REPLACE_WITH_EXTENSIONS_8_VERSION!!', '$(MicrosoftExtensionsHttpResiliencePackageVersionForNet8)')
-                                                 .Replace('!!REPLACE_WITH_EXTENSIONS_9_VERSION!!', '$(MicrosoftExtensionsHttpResiliencePackageVersionForNet9)'))"
+                                                 .Replace('!!REPLACE_WITH_EXTENSIONS_8_VERSION!!', '$(MicrosoftExtensionsHttpResiliencePackageVersion)')
```

Comment:
NIT: We should probably just change the templates so they remove this and swap it for the 9 version only.

Reply:
Yeah, calling it `REPLACE_WITH_EXTENSIONS_8_VERSION` and putting version 9 into it is wrong

Reply:
Maybe `REPLACE_WITH_DOTNET_EXTENSIONS_VERSION`

Reply:
Fixed.